docs/source/features/opmlimport.md

```diff
@@ -0,0 +1,12 @@
+OPML file import
+================
+RSS Guard can import file on OPML format 
```

Comment:
Sorry the English is really not that good, but for this time I will merge and fix spelling. Next time make sure to improve your English in th upcoming PRs.